src/pack_tools.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+import os
+from os import mkdir, rename, listdir
+from os.path import join, basename, splitext
+from shutil import copy, make_archive
+
+
+def prepare_tool(output_dir, name, run_file, manifest_file, gpt_graph_file=None):
+    tool_dir = join(output_dir, name)
+    mkdir(tool_dir)
```

Comment:
shouldn't you check first if the directory you're creating already exists?

Reply:
This is not responsibility of the `wings_shipper`